package oxberrypis.net.proto.rpi;

message StockEvent {
  required int32 stock_id = 1;
  required int32 timestamp = 2;
  required int32 pi_id = 3;
  required int32 seq_num = 4;
  optional int32 top_buy_price = 5;
  optional int32 top_sell_price = 6;
  optional int32 trade_price = 7;
}
